tools/individual-pkg-gen/copy-files-removing-deps.ts

```diff
@@ -0,0 +1,217 @@
+import * as path from 'path';
+import * as awsCdkMigration from 'aws-cdk-migration';
+import * as fs from 'fs-extra';
+// eslint-disable-next-line @typescript-eslint/no-require-imports
+const lerna_project = require('@lerna/project');
+
```

Comment:
I generally find this file a little hard to read and not so modular.Would be nice to clean it up and raise the bar on code quality.

(not a blocking comment)

Reply:
It's not only hard to read and not so modular - [it's dogshit](https://github.com/aws/aws-cdk/pull/15442#pullrequestreview-700542544) 😃.